src/app/component/modal/modal.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+import { useTranslations } from "next-intl";
+
+import { useConnect } from "wagmi";
+import { useAppDispatch, useAppSelector } from "@/redux/hooks";
+import WalletConnect from "@/app/svg/WalletConnect";
+import MetaMask from "@/app/svg/MetaMask";
+import CoinBase from "@/app/svg/CoinBase";
+import { toggleModal } from "@/redux/features/auth-slice";
+
+const Modal = ({}) => {
```

Comment:
This name is not correct, is too specific to be called Modal, can be called LoginModal for example

Reply:
fixed